src/hip_hcc.cpp

```diff
@@ -2460,3 +2461,28 @@ hipError_t hipHccGetAcceleratorView(hipStream_t stream, hc::accelerator_view** a
 
 //// TODO - add identifier numbers for streams and devices to help with debugging.
 // TODO - add a contect sequence number for debug. Print operator<< ctx:0.1 (device.ctx)
+
+namespace hip_impl {
+    std::vector<hsa_agent_t> all_hsa_agents() {
```

Comment:
The caller of this function is in a header and it could be problematic to some user environment if the header and the source have a different implementation of std::vector for some reason...

Reply:
You have more problems than that if for some reason you are mixing implementations of std::vector...